prime-router/src/main/kotlin/serializers/Hl7Serializer.kt

```diff
@@ -467,15 +564,23 @@ class Hl7Serializer(val metadata: Metadata) {
         terser.set("SFT-6", buildDate)
 
         terser.set("/PATIENT_RESULT/PATIENT/PID-1", "1")
+        terser.set("/PATIENT_RESULT/PATIENT/PID-3-4-3", "ISO")
 
```

Comment:
I don't think this is a good idea. To add sub-field without checking if the field is present, seems like it will lead to other problems. Isn't there another way?